application-templates/starter/src/components/entry-point/entry-point.js

```diff
@@ -55,6 +56,7 @@ class EntryPoint extends React.Component {
         DEV_ONLY__loadNavbarMenuConfig={() =>
           import('../../../menu.json').then(data => data.default || data)
         }
+        featureFlags={FEATURE_FLAGS}
```

Comment:
This trick actually helps so that custom apps don't get any flags at all. Otherwise the whole point would defeated. Regardless, custom apps being out there receiving all flags will kind of delay the helpfulness of this.

Reply:
Nice!